README.md

```diff
@@ -135,6 +139,13 @@ Here's my setup for toggleterm, including an autocmd to automatically close a gi
             open = "alternate"
         },
         callbacks = {
+            should_block = function(argv)
+                -- In this case, we would block if we find the diff-mode option
+                -- Note that argv contains all the parts of the CLI command, including
+                -- Neovim's path, commands, options and files.
+                -- See: :help v:argv
+                return vim.tbl_contains(argv, "-d")
```

Comment:
Maybe use `-b` here for a more general blocking config recipe, like you suggested originally? I'd like to implement custom handling for `-d` so that users can easily choose a diff viewer to open and customize behavior for diffs, so I'd rather wait to document that feature.